ceph.spec.in

```diff
@@ -495,7 +495,7 @@ See the Dashboard documentation at http://docs.ceph.com/ for details and a
 detailed feature overview.
 
 %package mgr-diskprediction-local
-Summary:        ceph-mgr diskprediction_local plugin
+Summary:        Ceph Manager plugin for predicting disk failures
```

Comment:
```suggestion
Summary:        Ceph Manager plugin for predicting disk failures locally
```
?

Reply:
@sebastian-philipp I considered that initially, but decided not to include that word because it makes the summary confusing. As a non-expert person reading the description, the word "locally" raises questions but provides no answers. The main question it raises is: "how else but locally?" IMO the summary should not try to provide all relevant information. The more subtle points can be left to the description, which can be much longer.

In order to really understand what "locally" means in this context, we would have to describe the cloud-based plugin (which uses a Google API to access Google's disk failure database?) but to me that's beyond the scope of a one-line summary.